versions/schematic-2.md

```diff
@@ -0,0 +1,235 @@
+<a name="tileEntityPosX"></a>x | `NBT int` | **Required**. As specified by chunk format above, BUT this value must to be converted to relative coordinates of the area. Relative to the `[0, 0, 0]` position of the schematic (without the [offset](#schematicOffset) applied).
+<a name="tileEntityPosY"></a>y | `NBT int` | **Required**. As specified by chunk format above, BUT this value must to be converted to relative coordinates of the area. Relative to the `[0, 0, 0]` position of the schematic (without the [offset](#schematicOffset) applied).
+<a name="tileEntityPosZ"></a>z | `NBT int` | **Required**. As specified by chunk format above, BUT this value must to be converted to relative coordinates of the area. Relative to the `[0, 0, 0]` position of the schematic (without the [offset](#schematicOffset) applied).
+<a name="tileEntityAdd"></a>{additions} | `<?>` | **Optional.** May be expanded by custom values needed for modifications. The corresponding Mod should at least be listed as optional within the [Metadata object](#metadataObject).
+
+##### Tile Entity Object Example
+
+An example of possible storage of a sign. See the [Minecraft Chunk Format](http://minecraft.gamepedia.com/Chunk_format#Block_entity_format) for a complete listing of data used to store various types of tile entities present in vanilla minecraft. Mods may store additional data or have additional types of tile entities.
+
+```NBT
+{
+    x: 0,
+    y: 1,
+    z: 2,
+    id: "minecraft:sign",
+    Text1: "foo",
+    Text2: "",
+    Text3: "bar",
+    Text4: ""
+}
+```
+
+#### <a name="entityObject"></a>Entity Object
+
+An object to specify all entitys which are within the area. Entities are used by Minecraft for movable or interactable objects (such as mobs or minecarts). The fields used to describe a entity vary for each type, however the structure will be the same as used by the [Minecraft Chunk Format](https://minecraft.gamepedia.com/Chunk_format#Entity_format).
+
+##### Fields
+
+Field Pattern | Type | Description
+---|:---:|---
+<a name="entityProp"></a>{properties} | `<?>` | **Required**. Amount and format of fields depending on the entity, specified by Mojang. As listed within the [Minecraft Chunk Format](https://minecraft.gamepedia.com/Chunk_format#Entity_format).
+<a name="entityPos"></a>Pos | `NBT list` of `NBT double`[3] | **Required.** As specified by chunk format above, BUT this value must to be converted to relative coordinates of the area. Relative to the `[0.0, 0.0, 0.0]` position of the schematic (without the [offset](#schematicOffset) applied). Must contain exactly 3 double values. This shall be saved as [NBT](#formFile) tag list of [NBT](#formFile) tag double.
+<a name="entityUUIDHigh"></a>UUIDMost | `NBT long` | **Required.** As specified by chunk format above, BUT a implementation must check if the UUID is already present and if so, replace this one by another.
```

Comment:
UUIDLeast and UUIDMost are NBT long tags like any other tag saved to an entity.
When serializing part of a world and saving an entity, it will contain its UUID from the source server like this.
When deserializing it to a destination server, this UUID might already exist. Or if someone inserts the same schematic twice on a world, it will already exist. Mojang doesn't allow double UUIDs, they need to be unique.
When using specific deserialization methods of mojang, it will actually use the UUID given by UUIDLeast and UUIDMost.
To prevent UUID conflicts, we should mention it.

Maybe better to delete it completely... (specify tags must be removed by serialization)?